node/core/runtime-api/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "polkadot-node-core-runtime-api"
+version = "0.1.0"
+authors = ["Parity Technologies <admin@parity.io>"]
+edition = "2018"
+
+[dependencies]
+futures = "0.3.5"
+log = "0.4.8"
+sp-api = { git = "https://github.com/paritytech/substrate", branch = "master" }
```

Comment:
should we consider creating patches/cargo overrides in the workspace?

Reply:
why?

Reply:
Seems beyond scope of PR. Ideally we'd have this stuff on crates.io